blog: bknd (#42411)
supabase/supabase: apps/www/_blog/2026-02-03-bknd-joins-supabase.mdx

---
title: 'BKND joins Supabase'
description: 'Dennis Senn, creator of BKND, is joining Supabase to build a Lite offering for agentic workloads.'
author: paul_copplestone
imgSocial: bknd-joins-supabase/og.png
imgThumb: bknd-joins-supabase/thumb.png
categories:
  - company
date: '2026-02-03:10:00:00'
toc_depth: 2
---

We're excited to welcome Dennis Senn, creator of [BKND](https://bknd.io), to the Supabase team.

With this addition, we're bringing Dennis's experience building lightweight backends and developer-first tooling into our ecosystem. The focus isn't to immediately integrate BKND into Supabase. Instead, Dennis will be working on a Lite offering on Supabase, designed specifically for agentic workloads.

### Lite, but not rushed

> There's a lot of excitement around building agentic systems. The right shape for a "Lite" offering is still evolving and we want to get it right. This move is about Dennis's expertise and collaboration, not a rushed product announcement. We'll share more as the direction becomes clear. Some ideas include:

Author: ```suggestion
There's a lot of excitement around building agentic systems. The right shape for a "Supabase Lite" offering is still a work-in-progress and we want to get it right. This move is about Dennis's expertise and collaboration, not a product announcement. We'll share more as the development progresses. Some ideas include:
```

- A trimmed-down experience for sandboxes.
- Databases architectures for agentic workloads.
- Smaller, cheaper, simpler databases.

### Next Steps

> BKND will stay open source. We will work in a separate repo to explore how to best build a lightweight Supabase experience. Stay tuned as we integrate Dennis into our team and explore what's possible for a lightweight Supabase experience. We're excited for what's to come and grateful for your support as we continue to grow.

Author: ```suggestion
BKND will stay open source. We will work in a separate repo to explore how to best build a lightweight Supabase experience. We're excited for what's to come and welcome any ideas as we build.
```
